### PR TITLE
Create Qick_sorting.cpp

### PR DESCRIPTION
**Pull request by akshaysharma2277**
Algorithm for Quick sort in cpp14.

### CODE REVIEW

File: code/sorting/quick_sort/Qick_sorting.cpp [hunk 1]
```diff
@@ -0,0 +1,68 @@
+#include <iostream>
+ 
+using namespace std;
+ 
+void quick_sort(int[],int,int);
```
Comment (arnavb) [inline code/sorting/quick_sort/Qick_sorting.cpp]: vectors are better than raw arrays.

File: code/sorting/quick_sort/Qick_sorting.cpp [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+int partition(int[],int,int);
+ 
+int main()
+{
+    int a[50],n,i;
```
Comment (arnavb) [inline code/sorting/quick_sort/Qick_sorting.cpp]: This makes no sense. You are setting a max size for the array. If you want to ask the user for an array size, use dynamic allocation, or better yet, vectors.

**Comment by iattempt** [comment]
the quick sort file existed

**Comment by arnavb** [comment]
@AdiChat Please close!

**Comment by akshaysharma2277** [comment]
Closing pull request

**Pull request closed without merging**